airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -1,2 +1,29 @@
-// Initialize the editor with the div id and path to Popcorn Editor.
-PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+$(document).ready(function () {
+  function getSlug() {
+    var config = $('slug');
+    if (config.length) {
+      return config.data('url');
```

Comment:
When you remove `<slug>` and use `<div id="editor">` instead, change the name from `url` to `slug`.